config/nrfconnect/chip-module/Kconfig.mcuboot.defaults

```diff
@@ -52,6 +52,34 @@ config FPROTECT
     bool
     default y
 
+# nRF7002DK uses SPI NOR external flash
+if BOARD_NRF7002DK_NRF5340_CPUAPP
+
+config SPI
+    bool
+    default y
+
+config SPI_NOR
+    bool
+    default y
+
+choice SPI_NOR_SFDP
+	default SPI_NOR_SFDP_DEVICETREE
+endchoice
+
+config SPI_NOR_FLASH_LAYOUT_PAGE_SIZE
+    int
+    default 4096
+
+config MULTITHREADING
+    bool
+    default y
+
+endif
+
+# All boards beside nRF7002DK use QSPI NOR external flash
+if BOARD_NRF5340DK_NRF5340_CPUAPP || BOARD_NRF52840DK_NRF52840
```

Comment:
What about the dongle or customers' boards? Maybe better to simply use `!BOARD_NRF7002DK_NRF5340_CPUAPP`

Reply:
I wasn't sure what is the best option. On the other hand dongle and customer boards may not support QSPI either.

Reply:
You're right. Better not to set anything if we don't know what is supposed to be used.

Reply:
I changed it also in other places, as previously I wasn't consistent.